functions-files/script_keywords.py

```diff
@@ -0,0 +1,18 @@
+import os
```

Comment:
@jlole Do we need to keep this in the repo? I think now that we have added the keywords everywhere, it can be dropped?

Reply:
yes it can be dropped, I will remove it